apps/web-giddh/src/app/gst/filing/filing.component.html

```diff
@@ -1,113 +1,115 @@
+                                <reconcile [activeCompanyGstNumber]="activeCompanyGstNumber" [currentPeriod]="currentPeriod"
+                                    [selectedGst]="selectedGst" [localeData]="localeData" [commonLocaleData]="commonLocaleData"></reconcile>
+                            </div>
+                        </tab>
+
+                        <tab *ngIf="selectedGst === GstReport.Gstr1"
+                            [active]="selectedTab === 'pushToGstn' || selectedTabId === 1"
+                            (selectTab)="selectTab($event, true, 'pushToGstn')">
+                            <ng-template tabHeading>
+                                {{localeData?.filing?.tabs?.push_gstn}}
+                            </ng-template>
+                            <div *ngIf="selectedTab === 'pushToGstn' || selectedTabId === 1">
+                                <push-to-gstin [activeCompanyGstNumber]="activeCompanyGstNumber" [showHsn]="showHsn"
+                                    (backClicked)="handleBackButton()" [currentPeriod]="currentPeriod"
+                                    [selectedGst]="selectedGst" [localeData]="localeData" [commonLocaleData]="commonLocaleData"></push-to-gstin>
                             </div>
-                        </div>
-                    </tab>
+                        </tab>
+
+                        <!-- Assigned by meghna  -->
+                        <tab *ngIf="isCompany && selectedGst !== GstReport.Gstr2"
+                            (selectTab)="selectTab($event, true, localeData?.filing?.tabs?.file_return)">
+                            <ng-template tabHeading>
+                                {{localeData?.filing?.tabs?.file_return}}
+                            </ng-template>
+                            <div class="transactions-overview">
+                                <div class="gstr-transactions-summary mr-t4 mr-b4 text-center">
+
+                                    <div class="successful-file-detail" hidden>
+                                        <img src="assets/images/file-successful.svg">
+                                        <h3>{{localeData?.filing?.gst_filed}}</h3>
+                                        <p>{{localeData?.filing?.return_file_note}}</p>
+                                        <button class="btn btn-primary  btn-lg"
+                                            (click)="fileReturn = {isAuthenticate: true }">{{localeData?.filing?.return_file}}
+                                        </button>
+
+                                    </div>
+                                    <div class="gst-return-date">
+                                        <img src="assets/images/return-date.svg">
+                                        <!-- <h3>File your GST Return of April 2019 till date</h3> -->
+                                        <p>{{localeData?.filing?.clickto_proceed}}</p>
```

Comment:
Consider this change
```suggestion
                                        <p>{{localeData?.filing?.click_to_proceed}}</p>
```